math/FunMath.cpp

```diff
@@ -1 +1,13 @@
 #include "FunMath.h"
+#include <vector>
+
+
+std::vector<int> Vect_Mult(std::vector<int> vect, int n){
```

Comment:
- Add a function comment: what does it do, what are the parameters/return values
- Make sure to follow our style guide for functions NamedLikeThis